jobserver/models.py

```diff
@@ -108,44 +124,54 @@ def get_absolute_url(self):
         return reverse("job-detail", kwargs={"pk": self.pk})
 
     @property
-    def is_complete(self):
-        return self.status_code is None and self.completed_at
+    def is_completed(self):
+        if not self.is_finished:
+            return False
```

Comment:
Still confused by the co-occurrence of `completed` and `finished`.

How does it make sense to call a job `completed` if it has a `status_code` of `None`?

Reply:
Hopefully [this](https://github.com/opensafely/job-server/pull/180#discussion_r517295111) comment clears up the confusion!

I was under the impression that a successful Job had no `status_code`?  Your question makes me think I'm wrong there!  Is that what `status_code = 0` is for?

Reply:
See

https://github.com/opensafely/job-runner/blob/06271b23443966c4db5e1bc9a81504d39b9c88c8/jobrunner/main.py#L33-L121

A null status_code is evidence the job has not completed.